### DISCUSSION TITLE
Using cleanup option

### DISCUSSION THREAD

**Discussion by mikekenyon99** [RhetTbull/osxphotos#1180] [General]
Hi - I want to use the --cleanup option but does it by default exclude the osxphotos database?  

So currently I export to /photos/ and in that folder is the osxphotos_export.db file.  

According to the instructions if I use --cleanup it will delete everything in that folder.  Can I just check that I don't need to protect this file?

Thanks

Mike

**Comment by RhetTbull** [comment]
Hi Mike. No, it will not delete the export database nor any other files created by osxphotos (such as reports). --cleanup is primarily used for deleting old versions of files (for example you deleted a file you had previously exported so it's no longer part of the export set). If you have files in the `/photos` folder that were not created by osxphotos that you want to keep, you can use `--keep` or create a `.osxphotos_keep` file in the export directory. See `osxphotos help export keep` for more info.

**Reply by mikekenyon99** [reply]
Thanks for the quick reply - much appreciated.

**Comment by mikekenyon99** [comment]
Hi - Apologies, a post thought question.  If I use --update --cleanup will this update new and changed photos and remove photos that I have deleted from Photos?  I.e. Can I use these two options to synchronise my folder of photos outside of Mac Photos?

**Reply by oPromessa** [reply]
Yes! That's the idea of export with --update and --cleanup to have a sync'ed exported file structure of the Photos library. 

Consider also using --exiftool and --touch

If you export to a NAS server folder do consider placing the exportdb to a local folder for faster exporting/better performance using --exportdb
